src/main/groovy/com/stormpath/tck/oauth2/Oauth2IT.groovy

```diff
@@ -271,11 +273,14 @@ class Oauth2IT extends AbstractIT {
     @Test(groups=["v100", "json"])
     void oauthClientCredentialsGrantSucceeds() throws Exception {
 
+        def account = createTestAccount()
+        Thread.sleep(1000) // sleep to get around rate limiting
```

Comment:
If we must have a sleep does it need to be a whole second? Also: I think I can turn off rate limiting on the okta tenant.